etcdserver/api/v3rpc/grpc.go

```diff
@@ -31,6 +31,7 @@ import (
 const (
 	grpcOverheadBytes = 512 * 1024
 	maxStreams        = math.MaxUint32
+	maxSendBytes      = math.MaxUint32
```

Comment:
for sent size, maybe we also want to make it configurable eventually. then users can limit the size of outbound response. one client wont eat up all bandwidth through a single huge list/watch.